skill-tree /src/test/java/com/RDS/skilltree/SkillsIntegrationTests.java

```diff
@@ -1,137 +1,300 @@
 package com.RDS.skilltree;
 
+import com.RDS.skilltree.Skill.*;
+import com.RDS.skilltree.User.*;
 import io.restassured.RestAssured;
 import io.restassured.response.Response;
-import org.junit.Before;
-import org.junit.jupiter.api.Disabled;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.After;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.testcontainers.containers.MySQLContainer;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.UUID;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.Matchers.*;
 
+@Testcontainers
+@Slf4j
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class SkillsIntegrationTests {
+    private UserRepository userRepository;
+    private SkillRepository skillRepository;
+    private static MySQLContainer mysqlContainer;
+    private final UserService userService;
+    private final SkillsService skillsService;
+    private UserDTO user;
+    private SkillDTO skill;
```

Comment:
Do you need these at the class level?

Reply:
Hi @debanjanc01 these I was using for storing the data which we inserted into db before each test and using that data in the test.